appium-examples/Cucumber/emusim/features/step_definitions/comment_steps.rb

```diff
@@ -0,0 +1,14 @@
+Given /^I click on the comment box$/ do
+  commentInput = @driver.find_element(:id, "comments")
```

Comment:
Ruby variables are snake case not camel case

Reply:
changed